api/src/test/java/marquez/db/migrations/V44_2__BackfillAirflowParentRunsTest.java

```diff
@@ -1,108 +0,0 @@
-/*
- * Copyright 2018-2022 contributors to the Marquez project
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package marquez.db.migrations;
```

Comment:
Please provide some comment in PR for reviewer about why are these files deleted

Reply:
Java dto changed (`NamespaceRow`), and they don't match the state of db during those tests. Solution would be to either provide separate java classes or delete those tests.

Since the migration is not from few versions ago and we did not hear about any bugs during the migration, the decision was to delete the tests.

Reply:
thanks